frontend/lang/ellinika.json

```diff
@@ -246,7 +246,7 @@
     "t_frontmatter_synchronized": "Μεταδεδομένα συγχρονίστηκαν ✔\n\nΑυτές οι παράμετροι θα χρησιμοποιηθούν σε μελλοντικές εξαγωγές.",
     "t_frontmatter_delete_field": "Διαγραφή πεδίου",
     "t_frontmatter_add_field": "Προσθήκη πεδίου {{plus}}",
-    "t_frontmatter_add_author": "Προσθήκη συγγραφέα {{plus}}",
+    "t_frontmatter_add_author": "Προσθήκη συγγραφέως {{plus}}",
```

Comment:
This form is my favourite and genderless. https://el.wiktionary.org/wiki/%CF%83%CF%85%CE%B3%CE%B3%CF%81%CE%B1%CF%86%CE%AD%CF%89%CF%82